app.js

```diff
@@ -416,6 +422,6 @@ schedule.scheduleJob('0 0 * * *', function(){
 */
 
 // Start server
-http.createServer(app).listen(5000, 'localhost', function() {
+http.createServer(app).listen(4999, 'localhost', function() {
```

Comment:
This was mistakenly committed I think
```suggestion
http.createServer(app).listen(5000, 'localhost', function() {
```